OSASINFRA-3066: Added migration procedure for Kuryr SDN. (#54822)
openshift/openshift-docs: networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc

:_content-type: ASSEMBLY
[id="migrate-from-kuryr"]
= Migrating from the Kuryr network plug-in
include::_attributes/common-attributes.adoc[]
:context: migrate-from-openshift-sdn

> toc::[]

Reviewer: We should add some tech-preview banner here.

:_content-type: ASSEMBLY
[id="migrate-from-kuryr-sdn"]
= Migrating from the Kuryr network plugin to the OVN-Kubernetes network plugin
include::_attributes/common-attributes.adoc[]
:context: migrate-from-kuryr-sdn

toc::[]

> include::snippets/technology-preview.adoc[]

Reviewer 2: ```suggestion
:FeatureName: Migration from Kuryr to OVN-Kubernetes
include::snippets/technology-preview.adoc[]
```

> As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the Kuryr SDN network plugin.

Reviewer 2: ```suggestion
As the administrator of a cluster that runs on {rh-openstack-first}, you can migrate to the OVN-Kubernetes network plugin from the Kuryr SDN network plugin.
```

:_content-type: ASSEMBLY
[id="migrate-from-kuryr-sdn"]
= Migrating from the Kuryr network plugin to the OVN-Kubernetes network plugin
include::_attributes/common-attributes.adoc[]
:context: migrate-from-kuryr-sdn

toc::[]

:FeatureName: Migration from Kuryr to OVN-Kubernetes
include::snippets/technology-preview.adoc[]

As the administrator of a cluster that runs on {rh-openstack-first}, you can migrate to the OVN-Kubernetes network plugin from the Kuryr SDN network plugin.

> To learn more about OVN-Kubernetes, read xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes#about-ovn-kubernetes[About the OVN-Kubernetes network plug-in].

Reviewer 2: ```suggestion
To learn more about OVN-Kubernetes, read xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes#about-ovn-kubernetes[About the OVN-Kubernetes network plugin].
```
